src/mobility/use-operators.tsx

```diff
@@ -4,7 +4,7 @@ import {FormFactor} from '@atb/api/types/generated/mobility-types_v2';
 export const useOperators = () => {
   const {mobilityOperators} = useFirestoreConfiguration();
 
-  return (types: undefined | FormFactor | FormFactor[]) => {
+  const byFormFactor = (types: undefined | FormFactor | FormFactor[]) => {
```

Comment:
I think we might want to start to always wrap methods returned in hooks with `useCallback`. After the "freeze-research" I have become more aware of how many renders we have and how easy it is to do things wrong, even though the current usage doesn't demand `useCallback`.

This was more of a general comment and not really necessary to change now, as we probably have quite a lot of hooks not enforcing this.